templates/components/_footer.html.twig

```diff
@@ -56,14 +56,24 @@
             <ul class="list__links list__links--col list__links--no-decor">
                 <li>
                     <h4>
-                        <a href="{{ path('page_campus') }}" class="text--bold text--medium-small">Campus</a>
+                        <a href="{{ path('page_campus') }}" class="text--bold text--medium-small">Formation</a>
                     </h4>
                 </li>
                 <li>
-                    <a href="{{ path('page_campus') }}">Toutes les ressources</a>
+                    <a href="https://touspolitiques.fr" target="_blank" rel="noopener noreferrer">
+                        L'institut de formation <em>Tous Politiques !</em>
+                    </a>
+                </li>
+                <li>
+                    <a href="https://mooc.en-marche.fr" target="_blank" rel="noopener noreferrer">
```

Comment:
Should this link be `https://mooc.en-marche.fr/faire-de-son-coup-de-fourchette-un-acte-politique` instead?

When `mooc.en-marche.fr` will be updated with a new mooc, the label "Le MOOC sur l'alimentation" will still be relevant then

Reply:
Yes, you're right. I'll update it !